README.md

```diff
@@ -1,145 +1,22 @@
-# Helm Chart for deployment of WSO2 API Microgateway 
+# Kubernetes and Helm Resources for WSO2 API Microgateway 
```

Comment:
@Aaquiff shall we remove the `Kubernetes` part out of this since we do not maintain pure Kubernetes resources here?

Reply:
Done